src/Raven.Server/Documents/ETL/EtlLoader.cs

```diff
@@ -75,16 +75,18 @@ public void Initialize(DatabaseRecord record)
                     foreach (var process in toRemove)
                     {
                         processes.Remove(process);
-                        _uniqueNames.Remove(process.ConfigurationName);
+                        _uniqueConfigurationNames.Remove(process.ConfigurationName);
                     }
                 }
 
+                var ensureUniqueConfigurationNames = _uniqueConfigurationNames.ToHashSet();
```

Comment:
This is already a `HashSet`, but I _think_ that in this case you'll be using one that is using case _sensitive_ comparer, which can cause conflicts later.

Reply:
You're right. Fixed in https://github.com/ravendb/ravendb/pull/9026/commits/5a9f0ff0c6846effe215c3b8dade4e56b89ea6bf. 

I'm making a copy of the global `HashSet` for validation purposes to avoid adding names of ETL tasks during the validation which aren't going to be started on this node anyway.